libs/buttons/src/lib/button/button.component.scss

```diff
@@ -0,0 +1,20 @@
+.content{
+    display: inline !important;
+}
+.mat-raised-button {
+    color: white !important;
+    font-size: 1.1rem;
+    width: 17rem;
+    height: 3.5rem;
+}
+.mat-raised-button.mat-primary {
+    background-color: #36B37E !important;
+    border-radius: 8px;
+}
+button:disabled {
+    opacity: 0.3;
+    pointer-events: none;
+}
+.mat-progress-spinner {
+    stroke: #fff;
+}
```

Comment:
Please ensure that this button is responsive. Use flex or inline-flex on the ```:host{}``` selector.